sys/drv/uhci.c

```diff
@@ -464,15 +473,23 @@ static void uhci_init_frames(uhci_state_t *uhci) {
   out32(UHCI_FLBASEADDR, uhci_physaddr(uhci->frames));
 }
 
+static inline size_t uhci_align_pool_size(size_t size) {
+  size_t log = log2(size);
+  size_t asize = 1UL << log;
+  return (asize != size) ? (asize << 1) : asize;
+}
```

Comment:
It's useful function, I thought that we already have something like that defined im mimiker.h (or in similar place). Could you make it generic? I'm pretty sure that other subsystems can borrow that implementation in the future.

Reply:
Good point. It turns out that I already have a second use case for this function (#1323). I created a separate PR for this function (#1330).